src/components/Forms/MonthlyGoodUpdateForm.vue

```diff
@@ -46,102 +46,108 @@
+						</li>
+					</ul>
 				</div>
+				<div class="row align-middle">
+					<div class="columns">
+						<span>
+							Optional donation to support Kiva
+						</span>
+					</div>
 
-				<div class="small-6 medium-4 columns">
-					<label
-						class="show-for-sr"
-						:class="{ 'error': $v.form.donation.$invalid }"
-						for="amount"
-					>
-						Donation
-					</label>
-					<kv-currency-input
-						class="text-input"
-						id="donation"
-						v-model="form.donation"
-					/>
+					<div class="small-6 medium-4 columns">
+						<label
+							class="show-for-sr"
+							:class="{ 'error': $v.form.donation.$invalid }"
+							for="amount"
+						>
+							Donation
+						</label>
+						<kv-currency-input
+							class="text-input"
+							id="donation"
+							v-model="form.donation"
+						/>
+					</div>
 				</div>
-			</div>
-			<div class="row column align-middle">
-				<ul class="text-right validation-errors" v-if="$v.form.donation.$invalid">
-					<li v-if="!$v.form.donation.minValue || !$v.form.donation.maxValue">
-						Enter an amount of $0-$10,000
-					</li>
-				</ul>
-			</div>
-			<div class="row">
-				<div class="columns">
-					<strong>Total/month</strong>
+				<div class="row column align-middle">
+					<ul class="text-right validation-errors" v-if="$v.form.donation.$invalid">
+						<li v-if="!$v.form.donation.minValue || !$v.form.donation.maxValue">
+							Enter an amount of $0-$10,000
+						</li>
+					</ul>
 				</div>
+				<div class="row">
+					<div class="columns">
+						<strong>Total/month</strong>
+					</div>
 
-				<div class="small-6 medium-4 columns">
-					<strong
-						class="additional-left-pad-currency"
-					>{{ totalCombinedDeposit | numeral('$0,0.00') }}</strong>
+					<div class="small-6 medium-4 columns">
+						<strong
+							class="additional-left-pad-currency"
+						>{{ totalCombinedDeposit | numeral('$0,0.00') }}</strong>
+					</div>
+				</div>
+				<div class="row column">
+					<ul class="text-center validation-errors"
+						v-if="!$v.form.mgAmount.maxTotal || !$v.form.donation.maxTotal"
+					>
+						<li>
+							The maximum Monthly Good total is $10,000.<br>
+							Please try again by entering in a smaller amount.
+						</li>
+					</ul>
 				</div>
 			</div>
-			<div class="row column">
-				<ul class="text-center validation-errors"
-					v-if="!$v.form.mgAmount.maxTotal || !$v.form.donation.maxTotal"
-				>
-					<li>
-						The maximum Monthly Good total is $10,000.<br>
-						Please try again by entering in a smaller amount.
-					</li>
-				</ul>
-			</div>
-			<div class="row column text-center">
-				Select a category to focus your lending
-				<kv-dropdown-rounded
-					v-model="form.category"
-					class="group-dropdown"
-				>
-					<option
-						v-for="(option, index) in lendingCategories"
-						:value="option.value"
-						:key="index"
+			<div class="row align-middle">
+				<div class="column">
+					<strong>Your contribution will:</strong>
```

Comment:
No real changes to this component except some alignment changes, and this copy change, see PR screenshot